### PR TITLE
upd centraldesktop/stomp to 2.0

### LINKED ISSUE #4: Add support php-stomp 2.0
https://github.com/centraldesktop/php-stomp/releases/tag/2.0.0
released on 10 Feb 2015

### PR DESCRIPTION
**Pull request by Gemorroj**
cleanup code and upd "centraldesktop/stomp" to 2.0 version.
**BC**: replace `host` and `port` configuration to `uri`.
this fixes #4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,10 @@ web/bundles/*
 
 # Configuration files
 app/config/parameters.ini
+
+# composer
+composer.lock
+composer.phar
+
+# PhpStorm
+.idea
```
Comment (mcg-web) [inline .gitignore]: this should be done in your global gitignore file not in the project...

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -36,8 +36,7 @@ public function getConfigTreeBuilder()
                                 ->prototype('array')
                                     ->addDefaultsIfNotSet()
                                     ->children()
-                                        ->scalarNode('host')->defaultValue('localhost')->end()
-                                        ->scalarNode('port')->defaultValue(61613)->end()
+                                        ->scalarNode('uri')->defaultValue('localhost:61613')->end()
```
Comment (mcg-web) [inline DependencyInjection/Configuration.php:39]: this is not a necessary BC

File: composer.json [hunk 3]
```diff
@@ -10,13 +10,12 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
-        "centraldesktop/stomp": "1.0.*",
-	"symfony/symfony": "~2.5|~3.0"
+        "php": ">=5.3.3",
+        "centraldesktop/stomp": "2.0.*",
+	    "symfony/symfony": "^2.8|^3.0"
```
Comment (mcg-web) [inline composer.json]: extra space

**Review by mcg-web — CHANGES_REQUESTED** [review]
Thank you :+1: ! some minor changes to be ready to merge...

File: DependencyInjection/Configuration.php (hunk 2 above)
Comment (Gemorroj) [reply DependencyInjection/Configuration.php:39]: I think `uri` is correct for greater similarity with centraldesktop/php-stomp (https://github.com/centraldesktop/php-stomp/blob/45a2fa09c6a70fb3c82c8964a3f233d59b2c4205/src/CentralDesktop/Stomp/ConnectionFactory/Simple.php#L8)

Comment (mcg-web) [reply DependencyInjection/Configuration.php:39]: ok

File: composer.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "overblog/activemq-bundle",
+    "name": "gemorroj/activemq-bundle",
```
Comment (mcg-web) [inline composer.json]: Don't forget to rollback this please :)

Comment (Gemorroj) [reply composer.json]: oh, sorry =)

**Review by mcg-web — APPROVED** [review]
Thank you!

**Pull request merged**